packages/tempus/src/Tempus_IntegratorAdjointSensitivity_impl.hpp

```diff
@@ -451,7 +451,9 @@ void IntegratorAdjointSensitivity<Scalar>::
 setObserver(Teuchos::RCP<IntegratorObserver<Scalar> > obs)
 {
   state_integrator_->setObserver(obs);
-  adjoint_integrator_->setObserver(obs);
```

Comment:
@etphipp Am I missing something here?  It looks like the observer is still being passed to the adjoint integrator.

Reply:
Look closer at the change.  The original line was commented out.

Reply:
Ugh.  Missread the diff.  :(